lib/presentation/search_page/components/autocomplete.dart

```diff
@@ -91,7 +94,11 @@ class _AutocompletePredictionsState extends State<AutocompletePredictions> {
                       onPressed: () async {
                         _getCurrentLocation();
 
-                        Hive.box(favCity).values.toList().forEach((element) {
+                        Hive
+                            .box(favCity)
+                            .values
+                            .toList()
+                            .forEach((element) {
```

Comment:
Maybe let's extract it to another variable and use shortcut in builds because it looks not to good :) and you're using it more than once.